src/state/groups/selectors.js

```diff
@@ -2,16 +2,77 @@ import { createSelector } from 'reselect';
 import { getLocale } from 'state/locale/selectors';
 
 export const getGroups = state => state.groups;
-
 export const getGroupsIdList = state => state.groups.list;
-
 export const getGroupsMap = state => state.groups.map;
+export const getGroupsTotal = state => state.groups.total;
```

Comment:
use createSelector

Reply:
Actually, in this case it's not needed, it even causes more bad than good.
The selector as is, makes a simple access to an object that will always be populated, so it's computationally very cheap, and there is no calculation.
If you use createSelector, you'll create a memoized selector with a single result cache, meaning that you have some disadvantages:
1. you will call one function more each time you access the selector: you have to call the selector function, plus the wrapper function that checks the arguments and either returns the memoized or recalculates the result
2. you will create a cache, that's basically a constantly synchronized copy of state.groups.total, and a waste of memory
3. the code will be more verbose and less readable.

In these cases, we should use memoized selectors only when there is an access to a state from another domain, in that case using createSelector makes sense because in case of refactoring, it would be less error prone